manual_flagging/yaml/inputs_update.yaml

```diff
@@ -2,19 +2,12 @@
 # We use these inputs when we want to re-flag already flagged sales.
 ---
 stat_groups:
-  single_family:
+  condos:
```

Comment:
**[Suggestion, non-blocking]** My understanding is that in the future, this data structure will have two top-level keys, `res` and `condo`; I wonder if we could switch to that data structure now so that we can avoid deleting the existing res stat groups?

Reply:
Related to this comment - https://github.com/ccao-data/model-sales-val/pull/95#discussion_r1473333219, I could build a temporary config file with boolean switches in the `manual_update.py` to fix this, but I'm guessing it would eventually be removed from main anyways with https://github.com/ccao-data/model-sales-val/issues/89. 

I agree that this structure/way of updating is not very good, but iterating on this on the fly with a time constrain + the time required for our proposed design changes make it so that I don't see a super good way to make an intermediate solution with staying power, happy to talk about this more though.